tf2_sensor_msgs/package.xml

```diff
@@ -20,7 +20,11 @@
 
   <build_export_depend>eigen</build_export_depend>
 
+  <depend>geometry_msgs</depend>
+  <depend>python3-numpy</depend>
   <depend>sensor_msgs</depend>
+  <depend>sensor_msgs_py</depend>
+  <depend>std_msgs</depend>
```

Comment:
These three new dependencies can all be `exec_depend`, I believe, since they are only ever used from Python.

(the geometry_msgs should be a `depend` since the C++ code is depending on it, we just missed it previously)